database/seeder/main.go

```
@@ -0,0 +1,191 @@
+        DisplayName:  fmt.Sprintf("%s User", name),
+        Email:        fmt.Sprintf("%s@test.com", name),
+        PasswordHash: "hashedpass",
+        PublicToken:  uuid.NewString(),
+        IsAdmin:      isAdmin,
+        VerifiedAt:   time.Now(),
+    }
+    db.Create(&user)
+    return user
+}
+
+func createPosts(db *gorm.DB, author database.User, count int) []database.Post {
```

golangci-lint (1.64.8)

func `createPosts` is unused

(unused)